src/protocols/kubernetes/settings.c

```diff
@@ -31,6 +31,8 @@ const char* GUAC_KUBERNETES_CLIENT_ARGS[] = {
     "namespace",
     "pod",
     "container",
+    "use-exec",
+    "exec-command",
```

Comment:
Why not just `exec-command`, defaulting to "attach" if no command is provided for execution?